codingtest_ajw/1일차/문자열-내-p와-y의-개수-찾기.js

```diff
@@ -0,0 +1,13 @@
+function solution(s) {
+    let p = 0;
+    let y = 0;
```

Comment:
너무 잘하셨는데 변수나 함수 네이밍 할때는 명확하게 해주시는게 좋습니다
```suggestion
    let countP = 0;
    let countY = 0;
```